07-self-diagnose for running azure-spring-cloud-in-vnet.md

```diff
@@ -0,0 +1,44 @@
+# Self-diagnose for Running Azure Spring Cloud in VNET
+
+##Navigate to the diagnostics page
+1. Sign in to the Azure portal.
+1. Go to your Azure Spring Cloud Overview page.
+1. Open Diagnose and solve problems in the menu on the left side of the page.
+1. Select the third category named **Networking**.
+
+## Execute the detector
```

Comment:
Users needn't to know about the concept `detector`

Reply:
Here I am following the typo with https://docs.microsoft.com/en-us/azure/spring-cloud/spring-cloud-howto-self-diagnose-solve. I think we can discuss with them later whether they will modify the description.